docs/tutorial/mysql.md

```diff
@@ -0,0 +1,99 @@
+# Connect Streamlit to MySQL
+
+## Introduction
+
+This guide explains how to securely access a MySQL database from Streamlit sharing or Streamlit for Teams. It uses the [mysql-connector-python](https://github.com/mysql/mysql-connector-python) library and Streamlit's [secrets management](../deploy_streamlit_app.html#secrets-management).
+
+## Create a MySQL database
+
+```eval_rst
+.. note:: If you already have a database that you want to use, feel free to `skip to the next step <mysql.html#add-username-and-password-to-your-local-app-secrets>`__.
+```
+
+First, follow [this tutorial](https://dev.mysql.com/doc/mysql-getting-started/en/) to install MySQL and start the MySQL server (note down the username and password!). Once your MySQL server is up and running, connect to it with the `mysql` client and enter the following commands to create a database and a table with some example values:
+
+```mysql
+CREATE DATABASE pets;
+
+USE pets;
+
+CREATE TABLE mytable (
+    name            varchar(80),
+    pet             varchar(80)
+);
+
+INSERT INTO mytable VALUES ('Mary', 'dog'), ('John', 'cat'), ('Robert', 'bird');
+```
+
+## Add username and password to your local app secrets
+
+Your local Streamlit app will read secrets from a file `.streamlit/secrets.toml` in your app's root directory. Create this file if it doesn't exist yet and add the database name, user, and password of your MySQL server as shown below:
+
+```python
+# .streamlit/secrets.toml
+
+[mysql]
+host = "localhost"
+port = 3306
+database = "xxx"
+user = "xxx"
+password = "xxx"
+```
+
+```eval_rst
+.. important:: Add this file to ``.gitignore`` and don't commit it to your Github repo!
+```
+
+## Copy your app secrets to the cloud
+
+As the `secrets.toml` file above is not committed to Github, you need to pass its content to your deployed app (on Streamlit sharing or Streamlit for Teams) separately. Go to the [app dashboard](https://share.streamlit.io/) and in the app's dropdown menu, click on **Edit Secrets**. Copy the content of `secrets.toml` into the text area. More information in [Secrets Management](../deploy_streamlit_app.html#secrets-management).
```

Comment:
In the last sentence of this paragraph, it's a bit terse. "More information is available at...." maybe?

Reply:
I've made the change to "More information is available at [Secrets Management]()." and reflected them in all the database tutorials.